alacritty.json

```diff
@@ -1,25 +1,21 @@
 {
     "homepage": "https://github.com/jwilm/alacritty",
     "description": "A cross-platform, GPU-accelerated terminal emulator",
-    "license": {
-        "identifier": "Apache-2.0",
-        "url": "https://www.apache.org/licenses/LICENSE-2.0"
-    },
+    "license": "Apache-2.0",
     "version": "0.2.4",
-    "url": [
-        "https://github.com/jwilm/alacritty/releases/download/v0.2.4/Alacritty-v0.2.4.exe#/alacritty.exe",
-        "https://github.com/jwilm/alacritty/releases/download/v0.2.4/winpty-agent.exe"
-    ],
-    "hash": [
-        "a31cdb56170f4973db42736f0e35d11b9ea43ab6729e3ab2fec5dc72b09cdc70",
-        "9add1a61155ec47cf6f347faf776b746eebbde1dc9360d81b8a909da34650642"
-    ],
-    "checkver": "github",
+    "url": "https://github.com/jwilm/alacritty/releases/download/v0.2.4/Alacritty-v0.2.4.exe#/alacritty.exe",
+    "hash": "a31cdb56170f4973db42736f0e35d11b9ea43ab6729e3ab2fec5dc72b09cdc70",
+    "depends": "extras/winpty",
+    "bin": "alacritty.exe",
+    "post_install": "Copy-Item \"$(appdir winpty)\\current\\x64\\bin\\winpty-agent.exe\" \"$dir\\winpty-agent.exe\"",
     "shortcuts": [
         [
             "alacritty.exe",
-            "Alacritty",
-            "--working-directory %USERPROFILE%"
```

Comment:
I think this should be added back in, because Alactritty would by default start in the location where it is located `C:\Users\USER\scoop\apps\alacritty\current`, but it would be ideal if it started in `USERPROFILE`, just like launching `cmd`?

Reply:
@notlmn yeah, didnt' check the shortcut. It starts with `~\scoop\apps\alacritty\current`. From the cmd/ps it starts with the current directory. Added it back with https://github.com/lukesampson/scoop-extras/commit/0719f4df794ccab9adec0fdfb9abf6954abbdec7